README.md

```diff
@@ -17,30 +17,33 @@ rendering component.
 # Requirements
 
 [Grunt](http://gruntjs.com) is used to build the UI.  Ensure grunt is installed
-via `npm install -g grunt-cli`.
+via `sudo npm install -g grunt-cli`.
```

Comment:
in instances where the user runs `nvm` or has npm installed in a directory they have permissions on, sudo isn't necessary.  I would change this to "sudo might be required depending on the installation"